src/components/app/data/hooks/useIsAssignmentsOnlyLearner.js

```diff
@@ -29,7 +29,7 @@ export default function useIsAssignmentsOnlyLearner() {
     subscriptionLicense,
     licenseRequests,
     couponCodeRequests,
-    couponCodesCount: couponCodeAssignments.length,
+    couponCodesCount: couponCodeRedemptionCount,
```

Comment:
Bug fix. With the frontend performance refactor, we no longer considered multi-use codes (i.e., individual codes that have multiple redemptions available).